package.json

```diff
@@ -8,6 +8,7 @@
     "node": "^10.16.3"
   },
   "scripts": {
+    "abc": "tsc",
```

Comment:
Is this needed?

Reply:
Oh, woops, that was to show something to @kerzhner. Deleting...